packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -692,17 +695,34 @@ class DropdownMenuItem<T> extends _DropdownMenuItemContainer {
     Key? key,
     this.onTap,
     this.value,
+    this.enabled = true,
     required Widget child,
   }) : assert(child != null),
-       super(key: key, child: child);
+        assert(enabled || (!enabled && value == null), 'if enabled is false, value should be null'),
+        assert(enabled || (!enabled && onTap == null), 'if enabled is false, onTap should be null'),
+        super(key: key, child: child);
```

Comment:
```suggestion
       assert(enabled || (!enabled && value == null), 'if enabled is false, value should be null'),
       assert(enabled || (!enabled && onTap == null), 'if enabled is false, onTap should be null'),
       super(key: key, child: child);
```

Reply:
Is there a tool or some method to indent lines correctly or am I supposed to keep in mind the indentation while coding?